render/wlr_renderer.c

```diff
@@ -1,9 +1,17 @@
 #include <assert.h>
 #include <stdbool.h>
 #include <stdlib.h>
+#include <wlr/config.h>
+
+#if WLR_HAS_GBM_SUPPORT
```

Comment:
`WLR_HAS_GBM_SUPPORT` doesn't exist in this commit. It's introduced in a later commit.

Reply:
Ah, no, it's added in `features += { 'gbm-support': true }`.

I'd prefer to defer the GBM issue to a later commit.